.github/workflows/cmake-multi-platform.yml

```
@@ -5,8 +5,20 @@ name: CMake on multiple platforms
 on:
   push:
     branches: [ "main" ]
+    paths-ignore:
+      - '**/*.md'
+      - 'docs/**'
+      - '!docs/requirements/**'
+      - 'LICENSE'
+      - '.cursor/**'
   pull_request:
     branches: [ "main" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)